extscheduler/resources/views/weeklyTest.view.xml

```diff
@@ -1,6 +1,13 @@
 <view xmlns="http://labkey.org/data/xml/view" template="app" frame="none" title="Scheduler - Week View">
     <dependencies>
         <dependency path="clientapi_core"/>
-        <dependency path="extscheduler/css/weeklyview.css"/>
+<!--        <dependency path="extscheduler/css/weeklyview.css"/>-->
+        <dependency path="extscheduler/App/view/EventFormAmended.js"/>
+        <dependency path="extscheduler/App/view/infoPanelAmended.js"/>
+        <dependency path="extscheduler/App/view/SchedulerAmended.js"/>
+        <dependency path="extscheduler/App/view/NavigationBarAmended.js"/>
+        <dependency path="extscheduler/App/view/ViewportAmended.js"/>
+        <dependency path="extscheduler/App/view/ViewportControllerAmended.js"/>
```

Comment:
```suggestion
        <dependency path="extscheduler/App/view/ViewportController.js"/>
        <dependency path="extscheduler/App/view/ViewportControllerAmended.js"/>
        <dependency path="extscheduler/App/view/ViewportAmended.js"/>
```

ViewportControllerAmended extends ViewportController so it may be needed. Also the order dependencies load could be a factor. Since ViewportControllerAmended is used by ViewportAmended you should add that dependency first. Try this out and let me know if it clears anything up.